Android/README.md

```diff
@@ -17,12 +17,15 @@ We recommend you refer to the FAQ at https://faq.whatsapp.com/general/26000226 f
   app should list each pack separately and each pack must have its own
   affordance to add it to WhatsApp (do not try to create "add all packs"
   operations).
+* Sticker packs must contain either static or animated stickers, never a mix of both
 * Each sticker pack must have a minimum of 3 stickers and a maximum of 30
   stickers
 * Stickers must be exactly 512 x 512 pixels
 * Stickers will render on a variety of backgrounds, white, black, colored, patterned, etc. Test your sticker art on a variety of backgrounds. For this reason, we recommend you add a 8px #FFFFFF stroke to the outside of each sticker. See the sample PSD referenced at https://faq.whatsapp.com/general/26000226 for more details.
-* Stickers must be in the [WebP format](https://developers.google.com/speed/webp). Currently, animated WebP or animated stickers are not supported. See the section [Converting to WebP](#converting-to-webp) below for information on how to create WebP files.
-* Each sticker must be less than 100KB. See the section [Tips for Reducing File Size](#tips-for-reducing-file-size) below. 
+* Stickers must be in the [WebP format](https://developers.google.com/speed/webp). See the section [Converting to WebP](#converting-to-webp) below for information on how to create WebP files.
+* Each static sticker must be less than 100KB and each animated sticker must be less than 500KB. See the section [Tips for Reducing File Size](#tips-for-reducing-file-size) below.
+* Animated stickers must have frames with minimum duration of 8ms. Animation duration should be below 10 seconds total.
```

Comment:
On iOS, we check for equality as well, so I added "should be less than or equal to 10 seconds total". Same for the sticker sizes above.

Reply:
yeah, same on Android. I will update.